src/main/java/seedu/address/logic/parser/SetRsvpCommandParser.java

```diff
@@ -28,9 +26,19 @@ public SetRsvpCommand parse(String args) throws ParseException {
             // Parse the index (first part)
             Index index = ParserUtil.parseIndex(parts[0]);
 
+            // Ensure the second part starts with "s/"
+            if (!parts[1].startsWith("s/")) {
+                throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, SetRsvpCommand.MESSAGE_USAGE));
+            }
+
             // Parse the action (second part)
-            int action = Integer.parseInt(parts[1]);
-            RsvpStatus rsvpStatus;
+            String actionString = parts[1].substring(2);
+            int action;
+            try {
+                action = Integer.parseInt(actionString);
+            } catch (NumberFormatException e) {
+                throw new ParseException("Invalid RSVP action format: must be a number.");
```

Comment:
This input check is similar to the checking of RSVP status in the setrsvpcommand class. Could consider combining it to one message to reduce redundant checks.

Reply:
I think no need to combine